packages/next-swc/crates/next-core/src/next_client/context.rs

```diff
@@ -173,7 +173,16 @@ pub async fn get_client_module_options_context(
 
     let tsconfig = get_typescript_transform_options(project_path);
     let decorators_options = get_decorators_transform_options(project_path);
-    let mdx_rs_options = *next_config.mdx_rs().await?;
+    let enable_mdx_rs = if *next_config.mdx_rs().await? {
+        Some(
```

Comment:
Probably not need to revise PR if there aren't else to update, but fyi we have https://github.com/vercel/next.js/blob/canary/packages/next-swc/crates/next-core/src/transform_options.rs consolidates fns to building options for the transforms.